### DISCUSSION TITLE
Does Dagster remove asset files ?

### DISCUSSION THREAD

**Discussion by dridk** [dagster-io/dagster#16291] [Q&A]
Hi 

We are using Dagster to store data as parquet file on a local storage ( NAS ) encrypted using gocryptfs.

After making big changes on assets codes, folders   wipping ...  we notify that our files are gones. Don't worry, we are Testing only. 

I think it comes from gocryptfs but I wonder if Dagster removes files in some way ?

**Comment by sryza** [comment]
Hi @dridk - Dagster does not delete asset files.
